src/main/jte/layout.jte

```diff
@@ -143,12 +143,22 @@
 
                                         @endif
 
+
+                                        <li><a href="/">${context.localize("calendar")}</a></li>
+                                        <li><a href="/bookings">${context.localize("bookings.my_bookings")}</a></li>
+
+                                        @if(login.kind() == LoginStateModel.Kind.ADMIN)
+                                            <div class="divider"></div>
+                                            <li><a href="/admin/users">${context.localize("administration.user")}</a></li>
+                                        @endif
+
                                         <li><a href="/1">${context.localize("calendar")}</a></li>
                                         <li><a href="/1/bookings">${context.localize("bookings.my_bookings")}</a></li>
 
                                         <div class="divider"></div>
 
                                         <li><a class="btn btn-outline btn-primary" href="/1/bookings/new">${context.localize("bookings.new")}</a></li>
```

Comment:
Not exactly the right PR for this discussion but why are we hardcoding for room 1? at least make it a constant.